server/db/satnogs-07.psql

```diff
@@ -0,0 +1,22 @@
+-- Add foreign key on observations referenced to satellite
+
+-- Fill missing IDs, name column was droped earlier and we lost the data
+UPDATE observations
+SET sat_id = s.sat_id
+FROM satellites s
+WHERE observations.sat_id is NULL AND
+    UPPER(s.sat_name) = CASE
+        WHEN REPLACE(observations.thumbnail, '-', ' ') ILIKE '%NOAA 15%' THEN 'NOAA 15'
```

Comment:
I'm ok with this (and not having tests for the upgrade), but please make sure there's a DB backup done before this is deployed on production.

Reply:
Sure, maybe it should be part of standard update flow?

Reply:
I include backup in ```setup.py```.

Reply:
I was thinking that doing a manual backup just once would be enough, but the automated backup is even better! thanks for writing the code.

Reply:
Tried to run it and it failed:

```
Finished processing dependencies for satnogs-server==1.0
Traceback (most recent call last):
  File "setup.py", line 57, in <module>
    backup_database()
  File "setup.py", line 50, in backup_database
    "PGPASSWORD": db["password"]
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/subprocess.py", line 342, in check_call
    retcode = call(*popenargs, **kwargs)
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/subprocess.py", line 323, in call
    with Popen(*popenargs, **kwargs) as p:
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/subprocess.py", line 775, in __init__
    restore_signals, start_new_session)
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/subprocess.py", line 1522, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'pg_dump': 'pg_dump'
(venv) copernicus:server thomson$ which pg_dump
/usr/local/bin/pg_dump
```

The pg_dump is there and I can run it manually. Investigating the problem right now.

Reply:
Problem fixed and fix pushed. Please review. If you're ok with it, resolve this.